diff --git a/cmake/config.cmake b/cmake/config.cmake
index 9754385fa..a093f61fe 100644
--- a/cmake/config.cmake
+++ b/cmake/config.cmake
@@ -79,7 +79,7 @@ set(USE_METAL OFF)
 # - ON: enable Vulkan with cmake's auto search
 # - OFF: disable vulkan
 # - /path/to/vulkan-sdk: use specific path to vulkan-sdk
-set(USE_VULKAN OFF)
+set(USE_VULKAN ON)
 
 # Whether enable OpenGL runtime
 set(USE_OPENGL OFF)
@@ -130,7 +130,7 @@ set(USE_LLVM OFF)
 set(USE_BYODT_POSIT OFF)
 
 # Whether use BLAS, choices: openblas, atlas, apple
-set(USE_BLAS none)
+set(USE_BLAS openblas)
 
 # Whether to use MKL
 # Possible values:
