From 10c90f246c524cdb3dea109e7b530f565ef4bc20 Mon Sep 17 00:00:00 2001
From: Joshua Whitley <josh.whitley@autoware.org>
Date: Mon, 16 Nov 2020 10:35:04 -0600
Subject: [PATCH] Test commit.

---
 cmake/config.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/config.cmake b/cmake/config.cmake
index 9754385fa..126c90196 100644
--- a/cmake/config.cmake
+++ b/cmake/config.cmake
@@ -130,7 +130,7 @@ set(USE_LLVM OFF)
 set(USE_BYODT_POSIT OFF)
 
 # Whether use BLAS, choices: openblas, atlas, apple
-set(USE_BLAS none)
+set(USE_BLAS openblas)
 
 # Whether to use MKL
 # Possible values:
-- 
2.25.1

